lib/agent/actions/fileretrieval/index.js

```diff
@@ -17,55 +19,84 @@ var fs = require('fs'),
 var common = require('./../../common'),
     system = common.system,
     node_bin = path.join(system.paths.current, 'bin', 'node'),
-    run_as_user = common.system.run_as_user,
     logger = common.logger,
     os_name = process.platform.replace('darwin', 'mac').replace('win32', 'windows');
 
 var em,
     cp;
 
+var UPLOAD_SERVER = 'http://solid.preyproject.com/upload/upload';
```

Comment:
take the scheme and domain from prey.conf (whose default value is `https://solid.preyproject.com`) and define the upload endpoint (for example) as part of this file, that is, `var UPLOAD_SERVER = method_that_takes_url_from_config() + '/upload/upload'`